src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Runtimes/MissingSdkRuntimeDetector.cs

```diff
@@ -73,14 +70,22 @@ private async Task RegisterSdkRuntimeNeededInProjectAsync(ConfiguredProject proj
 
                 string? TargetFrameworkVersion = await configuration.TargetFrameworkVersion.GetDisplayValueAsync();
```

Comment:
nit: The two variables above start with a capital letter, but are method variables. That should start with a lowercase.